schedulers/base.go

```diff
@@ -252,179 +248,169 @@ func (s *BaseScheduler) StatusUpdate(driver sched.SchedulerDriver, status *mesos
 }
 
 func (s *BaseScheduler) LogSlaveLost(slaveID *mesos.SlaveID) {
-	lmt := elekLogDef.ERROR
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("SLAVE LOST: SlaveID = %s", slaveID)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.ErrorLevel,
+		log.Fields{"SlaveID": fmt.Sprintf("%s", slaveID)}, "SLAVE LOST")
 }
 
 func (s *BaseScheduler) LogExecutorLost(executorID *mesos.ExecutorID, slaveID *mesos.SlaveID) {
-	lmt := elekLogDef.ERROR
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("EXECUTOR LOST: ExecutorID = %s, SlaveID = %s", executorID, slaveID)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.ErrorLevel,
+		log.Fields{"ExecutorID": fmt.Sprintf("%s", executorID), "SlaveID": fmt.Sprintf("%s", slaveID)}, "EXECUTOR LOST")
 }
 
 func (s *BaseScheduler) LogFrameworkMessage(executorID *mesos.ExecutorID,
 	slaveID *mesos.SlaveID, message string) {
-	lmt := elekLogDef.GENERAL
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("Received Framework message from executor [%s]: %s", executorID, message)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.InfoLevel,
+		log.Fields{"Received Framework message from executor": executorID}, message)
 }
 
 func (s *BaseScheduler) LogMesosError(err string) {
-	lmt := elekLogDef.ERROR
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("MESOS ERROR: %s", err)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.ErrorLevel,
+		log.Fields{"MESOS CONSOLE": fmt.Sprintf("%v", err)}, "")
 }
 
 func (s *BaseScheduler) LogElectronError(err error) {
-	lmt := elekLogDef.ERROR
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("ELECTRON ERROR: %v", err)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.ErrorLevel,
+		log.Fields{"ELECTRON CONSOLE": fmt.Sprintf("%v", err)}, "")
 }
 
 func (s *BaseScheduler) LogFrameworkRegistered(frameworkID *mesos.FrameworkID,
 	masterInfo *mesos.MasterInfo) {
-	lmt := elekLogDef.SUCCESS
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("FRAMEWORK REGISTERED! frameworkID = %s, master = %s",
-		frameworkID, masterInfo)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.InfoLevel,
+		log.Fields{"frameworkID": fmt.Sprintf("%s", frameworkID), "master": fmt.Sprintf("%s", masterInfo)}, "FRAMEWORK REGISTERED!")
 }
 
 func (s *BaseScheduler) LogFrameworkReregistered(masterInfo *mesos.MasterInfo) {
-	lmt := elekLogDef.GENERAL
-	msgColor := elekLogDef.LogMessageColors[lmt]
-	msg := msgColor.Sprintf("Framework re-registered with master %s", masterInfo)
-	s.Log(lmt, msg)
+	lmt := elekLogTypes.CONSOLE
+	elekLog.ElektronLog.Log(lmt,
+		log.InfoLevel,
+		log.Fields{"master": fmt.Sprintf("%s", masterInfo)}, "Framework re-registered")
```

Comment:
type mismatch. You might want to use either %v or you can use [masterInfo.GetHostname()](https://github.com/mesos/mesos-go/blob/29de6ff97b48c29cb5ac07029ed75186e5ba0eed/api/v0/mesosproto/mesos.pb.go#L1581).

Reply:
Done